pkgs/top-level/all-packages.nix

```diff
@@ -3909,6 +3909,8 @@ with pkgs;
 
   facter = callPackage ../tools/system/facter { };
 
+  factplusplus = callPackage ../applications/science/logic/factplusplus { };
```

Comment:
Would you mind moving the package to [`by-name`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md)?